src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -29,6 +29,12 @@
   <PropertyGroup Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true' and '$(TargetsBrowser)' != 'true'">
     <RuntimeSpecificFrameworkSuffix>.Mono</RuntimeSpecificFrameworkSuffix>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(MonoEnableLLVM)' == 'true'">
```

Comment:
this should include the condition from above (RuntimeFlavor+TargetsMobile+TargetsBrowser)

Reply:
RuntimeFlavor, sure, but we explicitly don't want to do this for Mobile/Browser (or explicitly _do_ want it for non-Mobile/Browser)

Reply:
@directhex exactly, that's why the condition does `'$(TargetsMobile)' != 'true'` :)

Otherwise we'd get the suffix if we e.g. pass `MonoEnableLLVM` on iOS.